src/index.js

```diff
@@ -3,30 +3,30 @@ const fs = require("fs");
 const path = require("path");
 const webpack = require("webpack");
 const MemoryFS = require("memory-fs");
-const WebpackParser = require('webpack/lib/Parser');
-const webpackParse = WebpackParser.parse
+const WebpackParser = require("webpack/lib/Parser");
+const webpackParse = WebpackParser.parse;
 
 // overload the webpack parser so that we can make
 // acorn work with the node.js / commonjs semantics
 // of being able to `return` in the top level of a
 // requireable module
 // https://github.com/zeit/ncc/issues/40
-WebpackParser.parse = function (source, opts = {}) {
+WebpackParser.parse = function(source, opts = {}) {
   return webpackParse.call(this, source, {
     ...opts,
     allowReturnOutsideFunction: true
   });
-}
+};
 
-const SUPPORTED_EXTENSIONS = [".js", ".mjs", ".json"];
+const SUPPORTED_EXTENSIONS = [".ts", ".tsx", ".js", ".mjs", ".json"];
```

Comment:
Added this

Reply:
Do we want to support ".tsx" or just ".ts"? Just seems like then we should be supporting ".jsx" as well?

Reply:
@guybedford `.tsx` is officially supported by TypeScript, but `.jsx` is not (by either TS or Node)

Reply:
I understand that, it just seems a little inconsistent to me that jsx can be supported through TS only... I don't think Deno supports tsx for example.